generator/src/test/java/com/scottlogic/deg/generator/reducer/ConstraintReducerTest.java

```diff
@@ -719,8 +719,8 @@
 //            .collect(Collectors.toList()));
 //
 //        final Set<AtomicConstraint> constraints = SetUtils.setOf(
```

Comment:
This test is entirely commented out. Should it be removed, or all of the tests marked `@Ignore` or `@Disabled` rather than updating commented out code?

Reply:
I don't believe I commented this code out so I am unclear as to the reasoning behind it.

Reply:
Its super out of date, I have removed it.